Replicator/Replicator.cc

```diff
@@ -420,6 +420,11 @@ namespace litecore { namespace repl {
         _connectionState = state;
 
         _checkpointer.stopAutosave();
+        
+        if(connected()) {
+            // The ability to get this goes away after the connection is closed, so cache it!
+            _peerTLSCertificate = connection().peerTLSCertificate();
```

Comment:
Where does _peerTLSCertificate get set when it's not connected such as having a TLS related error?

Reply:
It is still connected at this level, even after failure.  This variable just means that the BLIP connection is still non-null.